examples/server/server.c

```diff
@@ -109,6 +109,10 @@ static struct group_info group_id_to_text[] = {
     static int devId = INVALID_DEVID;
 #endif
 
+#ifdef HAVE_ARIA
+    #include "aria-crypt.h"
```

Comment:
yes, but without it the code would not compile. we certainly don't want it on all the time?

Reply:
Still testing this out, but @gojimmypi can you elaborate on why it would not compile? Maybe slack me if the details are too long for here. This include and the additional ones added to aria-crypt.c and aria-cryptocb.c also caught my attention.

Reply:
Hi @JacobBarthelmeh I must have been mistaken. I revisited this today & confirmed (twice) that indeed the include is not actually needed.

Reply:
Checked the recent code, it looks like the include is still in multiple files. client.c for example.

Reply:
oh yes, my bad. I had addressed only the `server.c` reference above. I've removed the include from the other examples.

I've also updated the PR description above with a bit more detail on building and testing. 

See also [the new README_cmake.md](https://github.com/gojimmypi/wolfssl/blob/aria/README_cmake.md) file.